core/exception/case_compare_spec.rb

```diff
@@ -1,5 +1,39 @@
 require_relative '../../spec_helper'
 
 describe "SystemCallError.===" do
-  it "needs to be reviewed for spec completeness"
+  before :all do
+    @example_errno_class = Errno::EINVAL
+    @example_errno = @example_errno_class::Errno
+  end
+
+  it "???" do
+    Errno::EINVAL.should === Errno::EINVAL.new
+  end
```

Comment:
Oops.  Meant to wait to hear about this description.

Reply:
@eregon, can you add it here?

Reply:
`returns true for an instance of the same class` would be good